test/drenv/providers/lima/k8s.yaml

```diff
@@ -8,6 +8,9 @@
 
 # yamllint disable rule:line-length
 ---
+
+minimumLimaVersion: 2.0.0
```

Comment:
The README says we require Lima 2.0.1 or later, but sets minimumLimaVersion: 2.0.0. Better to align both to 2.0.1?